### DISCUSSION TITLE
Welcome to Redash's new fork, let's have a brand new start from another time and space

### DISCUSSION THREAD

**Discussion by Jengro777** [getredash/redash#6033] [General]
https://github.com/RedashCommunity/redash

Given that Redash cannot merge the code, it is necessary for us to continue from another location

**Comment by justinclift** [comment]
Um, did you cut-n-paste a bad URL?

The new Redash Community fork is at [RedashCommunity/redash](https://github.com/RedashCommunity/redash). :wink:

**Reply by Jengro777** [reply]
> 嗯，你剪切粘贴了一个错误的网址吗？
> 
> 新的 Redash Community 分叉位于 [RedashCommunity/redash](https://github.com/RedashCommunity/redash)。😉

I modified it to the correct website

**Comment by Jengro777** [comment]
Community/redash  deploy

**https://github.com/RedashCommunity/redash/discussions/26**
